aether-ui/aether/ui/assets/apps/pipeline/components/PipelineCard.jsx

```diff
@@ -59,11 +64,11 @@ class PipelineCard extends Component {
         <div className='preview-contracts'>
           {
             pipeline.contracts.map(contract => (
-              <ContractCard key={contract.id} contract={contract} />
+              <ContractCard key={contract.id} contract={contract} {...this.props} />
```

Comment:
`{...this.props}` is useless if you use the redux features...

Reply:
`this.props.history` is required for navigation downstream. It has bee refactored to pass just that.